readme_source.md

```diff
@@ -192,33 +225,92 @@ kfutil stores rot generate-template --type stores \
 ```
 
 With all the files generated and populated, you can now run the reconcile command:
+
 ```bash
 kfutil stores rot reconcile \
   --stores stores_template.csv \
   --add-certs certs_template.csv \
   --remove-certs certs_template2.csv
 ```
-This will generate an audit file that contains the results of the audit and actions will immediately execute those actions.
+
+This will generate an audit file that contains the results of the audit and actions will immediately execute those
+actions.
 By default, the reconcile file will be named `rot_audit.csv` and will be written to the current directory. To output
 the reconcile file to a different location, use the `--output` flag:
+
 ```bash
 kfutil stores rot reconcile \
   --stores stores.csv \
   --add-certs addCerts.csv \
   --remove-certs removeCerts.csv \
   --output /path/to/output/audit_file.csv
 ```
+
 Alternatively you can provide an audit CSV file as an input to the reconcile command using the `--import-csv` flag:
+
 ```bash
 kfutil stores rot reconcile \
   --import-csv /path/to/audit_file.csv
 ```
 
-### Development
+### Certificate Store Inventory
+
+For full documentation, see [stores inventory](docs/kfutil_stores_inventory.md).
+
+#### Show the inventory of a certificate store
+
+For full documentation, see [stores inventory show](docs/kfutil_stores_inventory_show.md).
+
+```bash
+# Show by store ID:
+```bash
+kfutil stores inventory show --sid <store id>
+
+# Nested command lookup: shows inventory of first cert store found
+kfutil stores inventory show \
+  --sid $(kfutil stores list | jq -r ".[0].Id")
+```
+
+Show by client machine name:
+
+```bash
+kfutil stores inventory show --client <machine name>
+
+# Nested command lookup: shows inventory of first cert store found
+kfutil stores inventory show \
+  --client $(kfutil orchs list | jq -r ".[0].ClientMachine")
+```
+
+#### Add certificates to certificate stores
+
+For full documentation, see [stores inventory add](docs/kfutil_stores_inventory_add.md).
+
+```bash
+# Add 2 certs to 2 certificate stores
+kfutil stores inventory add \
```

Comment:
Should we mention that the order of the parameters is important?  It should be kind of obvious.. but maybe?  Just a thought.

Reply:
I don't think the order matters with flags I think the only time order matters is if there is an arg and some CLIs don't care, others requires all flags to be after all args and there might even be some that require all flags to be set before all args.